server/util_test.go

```diff
@@ -30,6 +30,11 @@ func TestParseSize(t *testing.T) {
 	if pn := parseSize(n); pn != 12345678 {
 		t.Fatalf("Did not parse %q correctly, res=%d\n", n, pn)
 	}
+
+	n = []byte("12345invalid678")
+	if pn := parseSize(n); pn != -1 {
+		t.Fatalf("Should error on %q, res=%d\n", n, pn)
```

Comment:
You don't need the `\n` at the end. Same for the other that you added. I know that it is in the existing test code but we try not to add them in new code.